src/file.cr

```diff
@@ -568,7 +568,7 @@ class File < IO::FileDescriptor
     Crystal::System::File.link(old_path, new_path)
   end
 
-  # Creates a symbolic link at *new_path* to an existing file given by *old_path.
+  # Creates a symbolic link at *new_path* to an existing file given by *old_path*.
```

Comment:
Does `old_path` really need to exist and be a file?

Reply:
The function does let you create broken symlinks.

Reply:
More apt would be `src_path` + `dest_path`.

Reply:
Or `target` and `link_name` from `man ln`. But changing the argument names should probably be a separate PR (unless @Givralix want's to tackle this here).

Reply:
I think it's better if you do it.